e2e/cypress/integration/api/iam/projects_api_scanjob_ingestion.spec.ts

```diff
@@ -0,0 +1,239 @@
+        if (node.name === nodeName ) {
+          cy.request({
+            headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+            method: 'DELETE',
+            url: `/api/v0/nodes/id/${node.id}`,
+            body: {}
+          });
+        }
+      });
+    });
+
+    // delete scan job
+    cy.request({
+      headers: {
+        projects: ['*'],
+        'api-token': Cypress.env('ADMIN_TOKEN')
+      },
+      method: 'POST',
+      url: '/api/v0/compliance/scanner/jobs/search',
+      body: {
+        filters: [
+          { key: 'job_type', values: ['exec']},
+          { key: 'parent_job', values: ['']}
+        ],
+        page: 1,
+        per_page: 100,
+        sort: 'end_time',
+        order: 'DESC'
+      }
+    }).then((resp: Cypress.ObjectLike) => {
+      resp.body.jobs.forEach((job: any) => {
+        if (job.name === scanJobName ) {
+          cy.request({
+            headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+            method: 'DELETE',
+            url: `/api/v0/compliance/scanner/jobs/id/${job.id}`,
+            body: {}
+          });
+        }
+      });
+    });
+  });
+
+  it('when a project has a rule that matches a node\'s environment' +
+  ' successfully associates that node with the project', () => {
+
+    // Create a secret
+    cy.request({
+      headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+      method: 'POST',
+      url: '/api/v0/secrets',
+      body: {
+        'name': secretName,
+        'type': 'ssh',
+        'data': [
+          {'key': 'username', 'value': targetUser},
+          {'key': 'key', 'value': targetKey}
+        ],
+        'tags': []
+      }
+    });
+
+    // get the secret - do I need a secret?
+    cy.request({
+        headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+        method: 'POST',
+        url: '/api/v0/secrets/search',
+        body: {}
+    }).then((getSecretResponse: Cypress.ObjectLike) => {
+      const secretId = getSecretResponse.body.secrets[0].id;
+      // Create a node
+      cy.request({
+          headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+          method: 'POST',
+          url: '/api/v0/nodes',
+          body: {
+            'name': nodeName,
+            'manager': 'automate',
+            'target_config': {
+              'backend': 'ssh',
+              'host': targetHost,
+              'secrets': [
+                secretId
+              ],
+              'port': 22
+            },
+            'tags': [{ 'key': 'environment', 'value': environment }]
+          }
+      }).then((nodeCreateResponse: Cypress.ObjectLike) => {
+        const nodeID = nodeCreateResponse.body.id;
+
+        // Create a scan job for that node
+        cy.request({
+          headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+          method: 'POST',
+          url: '/api/v0/compliance/scanner/jobs',
+          body: {
+            'name': scanJobName,
+            'tags': [],
+            'type': 'exec',
```

Comment:
we wanted to add a full scan job test bc a report from a scan job is processed internally direct to compliance ingestion (it doesn't go through an endpoint)